crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1085,6 +1085,19 @@ spec:
                                 description: "TCP traffic is measured in sent/received/total bytes. Only request traffic supplies response codes. Value must be one of: `none`, `sent`, `received`, or `total`."
                                 type: string
                                 enum: ["none","sent","received","total"]
+                      list:
+                        description: "Default settings for the List views (Apps, Workloads, etc)."
+                          include_health:
+                            description: "Include Health column (by default) for applicable list views. Setting to false can improve performance."
+                            type: boolean
+                          include_istio_resources:
+                            description: "Include Istio resources (by default) in Details column for applicable list views. Setting to false can improve performance."
+                            type: boolean
+                          include_validations:
+                            description: "Include Configuration validation column (by default) for applicable list views. Setting to false can improve performance."
+                            type: boolean
+                          show_include_toggles:
+                            description: "If true list pages display checkbox toggles for the include options, Otherwise the configured settings are applied but can not be changed by the user. Default is false."
```

Comment:
```suggestion
                            description: "If true list pages display checkbox toggles for the include options, Otherwise the configured settings are applied but can not be changed by the user. Default is false."
                            type: boolean
```